### ISSUE TITLE
Templating Inside of Description for Create Finding Workflow Node

### ISSUE THREAD

**Issue by Rhynorater** [caido/caido#1305]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

It would be great if we could use references inside of the Description in Create Finding nodes:
![Image](https://github.com/user-attachments/assets/26d5b898-aad5-4cbb-b298-ce7288306a3e)

That is how it would look ^. See #1304 for an example of why we would want this.

**Comment by Sytten** [comment]
Yes we also need that to build the deduplication key, I think we will adopt the github convention of `{{ node_alias.property }}`

**Comment by Sytten** [comment]
This will be in the next release
